README.md

```diff
@@ -1,32 +1,37 @@
 # pySwissPairer
+
 Swiss Pairing Algorithm written in Python
 
 ## Algorithm Overview
 
 The goal is to create pairings for each round of a tournament such that players with more points are paired against other players with more points, and no one is paired against someone they've been paired against previously.
 
-This algorithm performs a depth-first search on a tree of potential pairings. Each node in the tree contains a pairing of two players, and a list of players that have been paired up until that point. 
+This algorithm performs a depth-first search on a tree of potential pairings. Each node in the tree contains a pairing of two players, and a list of players that have been paired up until that point.
 
 1. Start by creating the root `PairingTreeNode`, which has a empty pairing and an empty list of paired players.
 1. Sort this `PairingTreeNode`'s list of unpaired players by points, high to low.
-  * If there are no players to pair, then all pairings have been found and the pairings are returned.
+
+- If there are no players to pair, then all pairings have been found and the pairings are returned.
```

Comment:
Nice functional changes, but you broke the formatting here with newlines. The numbered list resets with the extras.

Reply:
edit: nevermind, it was already broken

Reply:
Yeah that's what I get for keeping a markdown "format on save" plugin enabled. The thing has a mind of its own.